charts/camunda-platform/values.yaml

```diff
@@ -24,6 +24,13 @@
 
 # Global configuration for variables which can be accessed by all sub charts
 global:
+  # number of regions that support the zeebe platform (2 for minimal multi-region setup)
```

Comment:
What do you mean by

> that support the zeebe platform

I find it confusing. Let's remove or reword, e.g. to something like:

> that this Camunda Platform instance is stretched across